docs/package.json

```diff
@@ -4,18 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "prettier": "prettier --write --config \"./.prettierrc\" \"**/*.{yml,json,html,css,js,md}\""
```

Comment:
Since everything's moving to the top level, I'm also replacing this with a script in order to use the same prettier version across the project.